python/src/spark_rapids_ml/knn.py

```diff
@@ -338,16 +338,31 @@ def _get_cuml_fit_func(self, dataset: DataFrame) -> Callable[  # type: ignore
         pass
 
     def write(self) -> MLWriter:
+        """Unsupported."""
         raise NotImplementedError(
             "NearestNeighbors does not support saving/loading, just re-create the estimator."
         )
 
     @classmethod
     def read(cls) -> MLReader:
+        """Unsupported."""
         raise NotImplementedError(
             "NearestNeighbors does not support saving/loading, just re-create the estimator."
         )
 
+    def save(self, path: str) -> None:
+        """Unsupported."""
+        raise NotImplementedError(
+            "ApproximateNearestNeighbors does not support saving/loading, just re-create the estimator."
```

Comment:
Seems need to replace "ApproximateNearestNeighbors" with "NearestNeighbors", as the function belongs to Nearestneighbors?